buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -2611,7 +2611,7 @@ class BeamModulePlugin implements Plugin<Project> {
             def distTarBall = "${pythonRootDir}/build/apache-beam.tar.gz"
             project.exec {
               executable 'sh'
-              args '-c', ". ${project.ext.envdir}/bin/activate && cd ${copiedPyRoot} && scripts/run_tox.sh $tox_env $distTarBall '$posargs'"
+              args '-c', ". ${project.ext.envdir}/bin/activate && cd ${copiedPyRoot} && scripts/run_tox.sh $tox_env $distTarBall $posargs"
```

Comment:
Why did we unquote this?

Reply:
There was a parsing error caused by this quotes, in the case that the string contains multiple quotes and double quotes, the solution was deleted this quotes. For example in python examples we had 
`"'apache_beam/examples/complete/game/ \
  apache_beam/examples/complete/autocomplete_test.py \
  apache_beam/examples/complete/distribopt_test.py \`

Gradlew failed to parse this string , failing the execution. 
The reason to execute all the test in one gradle job is to save time and resources. Also I tried using regular expressions but doesn't work , gradle parse it literally. 
Put in a matrix all the jobs generate a lot of executions only for one small test. 
 I tested this change with other workflows and everything was well. But I worried if this change could affect another code or tests.

Reply:
> I tested this change with other workflows and everything was well. But I worried if this change could affect another code or tests.

That's my concern here too. Can we just drop the `'` for python examples?

Reply:
Hi @damccorm  
I did the changes, the quotes get back in the file. I tested again the tests, and everything works well.